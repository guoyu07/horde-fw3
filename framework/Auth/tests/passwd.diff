001- array(1) {
001+ Fatal error: Class 'Horde' not found in /usr/share/php/Horde/Auth/passwd.php on line 93
002-   [0]=>
003-   string(4) "user"
004- }
005- bool(true)