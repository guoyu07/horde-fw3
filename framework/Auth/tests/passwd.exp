array(1) {
  [0]=>
  string(4) "user"
}
bool(true)