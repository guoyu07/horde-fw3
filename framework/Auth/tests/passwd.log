
---- EXPECTED OUTPUT
array(1) {
  [0]=>
  string(4) "user"
}
bool(true)
---- ACTUAL OUTPUT
Fatal error: Class 'Horde' not found in /usr/share/php/Horde/Auth/passwd.php on line 93
---- FAILED
