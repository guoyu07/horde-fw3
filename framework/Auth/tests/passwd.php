<?php

require_once 'Horde/Util.php';
require_once 'Horde/String.php';
require_once dirname(__FILE__) . '/../Auth.php';

$auth = Auth::factory('passwd', array('filename' => dirname(__FILE__) . '/test.passwd'));

// List users
var_dump($auth->listUsers());

// Authenticate
var_dump($auth->_authenticate('user', array('password' => 'password')));

?>
