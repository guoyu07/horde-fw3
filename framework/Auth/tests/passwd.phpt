--TEST--
Auth_passwd test
--FILE--
<?php

require_once 'Horde/Util.php';
require_once 'Horde/String.php';
require_once dirname(__FILE__) . '/../Auth.php';

$auth = Auth::factory('passwd', array('filename' => dirname(__FILE__) . '/test.passwd'));

// List users
var_dump($auth->listUsers());

// Authenticate
var_dump($auth->_authenticate('user', array('password' => 'password')));

?>
--EXPECT--
array(1) {
  [0]=>
  string(4) "user"
}
bool(true)
