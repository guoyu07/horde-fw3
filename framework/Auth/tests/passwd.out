Fatal error: Class 'Horde' not found in /usr/share/php/Horde/Auth/passwd.php on line 93